lib/bolus.js

```diff
@@ -90,6 +90,14 @@ function reduce (treatments) {
         annotate("Normal bolus (solo, no bolus wizard).");
       }
 
+      if (has_insulin) {
+	var myopenaps_dir = "/root/myopenaps/";
```

Comment:
We'll need to figure out another way to do this.  Hard-coding the path won't work for some rigs, and makes the tests fail.